python/ray/data/aggregate.py

```diff
@@ -86,15 +86,27 @@ def accumulate_block(a: AggType, block: Block) -> AggType:
                     a = accumulate_row(a, r)
                 return a
 
+        if _merge_batch is None:
```

Comment:
When would we not go down this code path?

Reply:
For custom user-implemented aggregations that still might be relying on existing API